src/Pixel.Identity.UI.Client/Pages/Account/Password.razor.cs

```diff
@@ -7,11 +7,11 @@
 
 namespace Pixel.Identity.UI.Client.Pages.Account
 {
-    public partial class Logins : ComponentBase
```

Comment:
Any reason to rename it given this page is used to manage external logins ? Should the file name be changed instead to Logins.* ?

Reply:
But it does not handle external logins, it shows the change password form.

Reply:
Simply put it allows to manage your logins. If there is no external logins setup, it will let you manage your password for local account login.